frontend/src/actions/events.js

```diff
@@ -1,72 +1,19 @@
 export const fetchEvents = () => {
   return dispatch => {
-    let headers = { "Content-Type": "application/json" };
-    return fetch("api/events", { headers })
-      .then(res => res.json())
-      .then(events => {
-        return dispatch({
-          type: "FETCH_EVENTS",
-          events
-        });
-      });
-  };
-};
-
-export const addEvent = event => {
-  return dispatch => {
-    let headers = { "Content-Type": "application/json" };
-    let body = JSON.stringify({ event });
-    return fetch("api/events/", {
-      headers,
-      method: "POST",
-      body
-    })
-      .then(res => res.json())
-      .then(event => {
-        return dispatch({
-          type: "ADD_EVENT",
-          event
-        });
-      });
-  };
-};
+    let headers = {
+      "Content-Type": "application/json",
+      Authorization: "Bearer KIFYHFTNBKFCIFOPNESJ"
```

Comment:
since this is just our key - its ok for now... but in the future we are going to want to leverage `.env`

https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables

Reply:
Thank you, Dan.  I will take care of the .env today